mmtrack/models/mot/qdtrack.py

```diff
@@ -0,0 +1,197 @@
+import numpy as np
+import torch
+from mmdet.core import bbox2result
+from mmdet.models import build_detector, build_head
+
+from mmtrack.core import imshow_tracks, outs2results, results2outs
+from mmtrack.models.mot import BaseMultiObjectTracker
+from ..builder import MODELS, build_tracker
+
+
+@MODELS.register_module()
+class QDTrack(BaseMultiObjectTracker):
+
+    def __init__(self,
+                 detector=None,
+                 track_head=None,
+                 tracker=None,
+                 freeze_detector=False,
+                 *args,
+                 **kwargs):
+        super().__init__(*args, **kwargs)
+        self.tracker_cfg = tracker
```

Comment:
why do you need to record `tracker_cfg`

Reply:
in the init_tracker() function, which would be called in the begining of every video to reset the tracklets.